paper-ieee-trans.tex

```diff
@@ -108,16 +108,16 @@ \section{Introduction}
 organized as follows. Section \ref{sec:ev} details the evolution of research on
 software reliability in EC-projects. Section \ref{sec:ntsr} provides
 information about new research trends in software reliability based upon the
-experiences from the recent {\sl INDIGO--DataCloud} project. Conclusions are drawn in Section \ref{sec:con}.
+experiences from the recent {\sl INDIGO--DataCloud} project (INDIGO--DC). Conclusions are drawn in Section \ref{sec:con}.
```

Comment:
This acronym is only used in the table, we could just put a footnote there..although I for me it is not even needed..

Reply:
let the acronym be there anyway
agree with change

Reply:
I am in favour of leaving the acronym as suggeste by Cristina without adding footnote. We can use if just in the table

Reply:
leaving acronym